modAionImpl/src/org/aion/zero/impl/pendingState/AionPendingStateImpl.java

```diff
@@ -1309,4 +1316,8 @@ public String getVersion() {
     public void updateBest() {
         getBestBlock();
     }
+
+    public void setPendingTxCallback(PendingTxCallback callback) {
```

Comment:
`AionHub` creates the pending state. Can this be moved to the constructor? Then we can ensure that the object is always non-null and no longer need the NPE checks.